server/views/macros/printLink.njk

```diff
@@ -0,0 +1,7 @@
+{% macro printLink(linkText = 'Print this page', align = "left", classes = "") %}
```

Comment:
Don't we already have this? I thought we made a macro to show the print link?

Reply:
Seems not. I looked for it as I thought I'd added it previously, but seems not.